EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -280,9 +303,9 @@ We can spaghetti plot generated case data from the version of the model _which h
 Because we are using synthetic data we can also plot the model predictions for the _unobserved_ infections and check that (at least in this example) we were able to capture some unobserved/latent variables in the process accurate.
 "
 
-# ╔═╡ e9df22b8-8e4d-4ab7-91ea-c01f2239b3e5
+# ╔═╡ e74fc652-cd5f-4764-a416-caa8bab0bf0c
 let
-    post_check_mdl = fix(full_epi_aware_mdl, (rw_init = 0.0,))
+    post_check_mdl = fix(full_epi_aware_mdl, (init_rw = 0.0,))
```

Comment:
can you remind me why we fix here?

Reply:
It is a non-identifiability between `DirectInfections` initialisation and the latent process (plus this is a typo I've just corrected which reveals a brittleness of needing to know the names of variables internally...)

In general we might want `rw_init` to represent the initial distribution of (say) $r_t$ or $\mathcal{R}_t$, and the initial incidence is a separate concept. But for log-transformed initial infections _and_ log-latent process we want one to be fixed.

NB: I actually didn't do that due to the typo and it all still ran so its not critical, but it will have been bad for the sampler.

Reply:
> In general we might want rw_init to represent the initial distribution of (say) 
 > or 
> , and the initial incidence is a separate concept. But for log-transformed initial infections and log-latent process > we want one to be fixed.

Yes I think we might want to think about it like this. Sounds like a new issue to me.

> NB: I actually didn't do that due to the typo and it all still ran so its not critical, but it will have been bad for the sampler.

Do you think this is a bug? It feels like a bug or at least a really bad developer choice. Worth an issue over on `DynamicPPL`?